src/Tests/dotnet-watch.Tests/LaunchSettingsProfileTest.cs

```diff
@@ -0,0 +1,147 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.DotNet.Watcher.Internal;
+using Microsoft.Extensions.Tools.Internal;
+
+namespace Microsoft.DotNet.Watcher.Tools;
+
+public class LaunchSettingsProfileTest
+{
+    private readonly IReporter _reporter;
+    private readonly TestAssetsManager _testAssets;
+
+    public LaunchSettingsProfileTest(ITestOutputHelper output)
+    {
+        _reporter = new TestReporter(output);
+        _testAssets = new TestAssetsManager(output);
+    }
+
+    [Fact]
+    public void LoadsLaunchProfile()
```

Comment:
Wouldn't a single test that has both a comment and comma be sufficient?

Reply:
I can combine them if you like - I just assumed more granular tests would be preferred.

Reply:
I think it would be better as a single test.